pkg/ddevapp/webserver_config_assets/nginx-site-drupal6.conf

```diff
@@ -1,4 +1,6 @@
 # ddev drupal6 config
+# https://www.drupal.org/project/drupal/issues/2937161
+# https://www.drupal.org/project/drupal/issues/3336659
```

Comment:
Our configs for drupal6/7 are probably as good as anything is going to get. These links aren't valid.

Reply:
```suggestion
```